applications/Bright_Disputes.md

```diff
@@ -0,0 +1,166 @@
+
+At the end of the voting phase, the Judge is responsible for counting all the votes and issuing a verdict. If there is no majority of the votes (at least 60% against one party), verdict can not be unequivocal. In that case the dispute round need to be repeated, but the number of juries is increated by two. What is worth to mention is that votes are not revealed, so Juries doesn't know how others vote. Thats why Judge need to provides a proof (zk-snark) of his correct counting of the votes. When there is a majority of votes, the Judge issues a verdict and make votes public. Now Judge and the Juries which were in the majority of the votes are rewarded by their work. Juries whose votes are in the minority are penalized, by slashing their escrows. If Judge or Jury will not fulfill his duties (take action in time), their will lose their escrows.
+
+Beneath is the diagram showing the basic logic of the Bright Disputes, which were here described.
+
+![](https://i.imgur.com/jWWw54D.png)
+
+
+More specifically, BrightDisputes would be used in connection with a job advertising platform. In the following diagram our own project SubFunds is used as an example of such platform. The diagram illustrates the whole use case scenario that involves a funding part that advertised the job (Bob), a contractor obliged to perform it (Alice) and a judge selected for the case (Charlie):
+
+![](https://i.imgur.com/aLGOXbJ.png)
+
+### Project Details
+
+"Bright Disputes" is a smart contract fully written in ink! From the development perspective, the most chalenging and interesting part will be the zk-snarks. Privacy is very important in our dispute system while the Juries publishes the encrypted votes, and the Judge counts them to reach a verdict. Role of the Judge requires honesty, he is the only one who can decrypt the votes. That's why we want to use zk-snarks to prove Judge honesty and correctness of the vote counting. Especially when the verdict is not clear and we need to repeat the voting round. The jury cannot see how others vote, otherwise the verdict will not be fair.
+
+Based on our knowlage of zk-snarks ([zk-snarks-in-substrate](https://brightinventions.pl/blog/zk-snarks-in-substrate-part-1/)), using this technology have some consequences. Creating of zk-snarks will generate some artifacts (verification / proving key). We will provide a CLI, which will simplify this process. As it shown on the diagram below Juries / Judge are going to use CLI to communicate with the smart contrant, while Owner / Contractor can use polkadot.js dApp.
+
+![](https://i.imgur.com/WjZczZx.png)
+
+Our target plaform for the deployment will be AlephZero, which provides zk-snarks verification.
+
+This project will not provide any UI. All the functionality can be tested by the polkadot.js dApp.
```

Comment:
We would like to see full stack dApps so that user's can interact with ink!. I see that your company has UI/UX developers. Is there a reason you are not using them? 

Please consider this and adjust the budget accordingly. You do not have to build a UI, but it would be preferred!